k8s/openebs-pre-release-features.yaml

```diff
@@ -230,6 +251,10 @@ spec:
               # From host, dir called /var/openebs/shared-<uid> is created to avoid clash if two replicas run on same node.
               path: /var/openebs/shared-a2b
               type: {{ .Config.HostPathType.value }}
+          - name: sparse
+            hostPath:
+              path: /var/openebs
+              type: Directory
```

Comment:
Can you change the type to : {{ .Config.HostPathType.value }}

Reply:
Yes I will do it.